internal/pkg/tail/tail.go

```diff
@@ -0,0 +1,92 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package tail
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+)
+
+// Window is the read window size for tail scanning.
+const Window = 4096
+
+// SeekLines seeks the passed io.ReadSeeker so that it's -N lines from the tail.
```

Comment:
couldn't find a better name for it, suggestions?

Reply:
I think this sounds fine. It describes what it does.